ticketguardian/affiliate/affiliate.py

```diff
@@ -18,3 +18,27 @@ def parent(self):
         if self._parent is not None and not hasattr(self._parent, 'resource'):
             self._parent = self._construct(obj=self._parent)
         return self._parent
+
+    @property
+    def scope(self):
+        """
+        Used to gets all children within the scope of this Affiliate.
+        Returns:
+            list: a list of all children of the Affiliate
+        """
+        return self.retrieve(self.id, 'scope', raw_data=True)
+
+    def get_parent_scope(self):
```

Comment:
Why is this one formatted as a `get_...` method while the other is a property? we should keep it consistent, I'm fine doing it either way

Reply:
Ok good point I just updated it right now.